pkg/server/v3/server_test.go

```diff
@@ -323,6 +484,58 @@ func TestResponseHandlers(t *testing.T) {
 	}
 }
 
+func TestDeltaResponseHandlers(t *testing.T) {
+	for _, typ := range testTypes {
+		t.Run(typ, func(t *testing.T) {
+			config := makeMockConfigWatcher()
+			config.deltaResponses = makeDeltaResponses()
+			s := server.NewServer(context.Background(), config, server.CallbackFuncs{})
+
+			resp := makeMockDeltaStream(t)
+			// This should put through a wildcard request since we aren't subscribing to anything
```

Comment:
`// This is a wildcard request since the list of subscriptions is empty` maybe?

Reply:
Much more concise